components/Header/Header.tsx

```diff
@@ -8,6 +8,7 @@ import { HeaderMobile } from "components/Header/components/HeaderMobile";
 export const links = [
 	{ name: "home", route: "/" },
 	{ name: "about", route: "/about" },
+	{ name: "company", route: "/company" },
```

Comment:
remove this. company page is not ready to be shown to users. we will add this when company page finishes